loadclient/sample_log_lines.go

```diff
@@ -0,0 +1,35 @@
+	"[localhost]: FAILED! => {\"msg\": \"An unhandled exception occurred while running the lookup plugin 'k8s'. Error was a <class 'ansible.errors.AnsibleError'>, original message: Failed to find exact match for kubevirt.io/v1.KubevirtCommonTemplatesBundle by [kind, name, singularName, shortNames]\"}",
+	"E0427 11:44:58.439709       1 memcache.go:206] couldn't get resource list for metrics.k8s.io/v1beta1: an error on the server",
+	"2021-04-27 11:39:43.113686 I | embed: rejected connection from \"1.1.1.1:37074\" (error \"EOF\", ServerName)",
+	"W0427 11:39:43.113579       1 clientconn.go:1208] grpc: addrConn.createTransport failed to connect to {https://1.1.1.1:2379  <nil> 0 <nil>}. Err :connection error: desc = transport: authentication handshake failed: context canceled. Reconnecting...",
+	"2021-04-27T09:24:51Z|00650|stream_ssl|WARN|SSL_write: system error (Connection reset by peer)",
+	"E0427 16:50:08.356649       1 status.go:71] apiserver received an error that is not an metav1.Status: &errors.errorString{s:\"context canceled\"}",
+	"E0427 20:06:06.185567       1 limiter.go:165] error reloading router: wait: no child processes",
+	"E0412 21:37:23.019649       1 proxy.go:73] Unable to authenticate the request due to an error: Post https://1.1.1.1:443/apis/authentication.k8s.io/v1/tokenreviews: context canceled",
+	"[ERROR] plugin/errors: 2 kubernetes.default.svc. A: read udp 1.1.1.1:54608->11.1.1.1:53: i/o timeout",
+	"E0426 20:39:28.065697       1 scheduler.go:599] error selecting node for pod: running \"VolumeBinding\" filter plugin for pod \"eric-data-document-database-pg-1\": pod has unbound immediate PersistentVolumeClaims",
+	"Warning: failed to query journal: Bad message (os error 74)",
+	"E0427 02:54:04.283531    8550",
+	"[DEBUG] plugin/errors: 2 kubernetes.default.svc. A: read udp 1.1.1.1:54608->1.1.1.1:53: i/o timeout",
+	"D0426 20:39:28.065697       1 scheduler.go:599] error selecting node for pod: running \"VolumeBinding\" filter plugin for pod \"eric-data-document-database-pg-1\": pod has unbound immediate PersistentVolumeClaims",
+}
```

Comment:
similarly here sample->system or sample->application  ?  We are  using simple_log_line also so from name its not clear.

Reply:
application :-) done